doc/settings.md

```diff
@@ -48,6 +50,23 @@ files from disk by specifying an `defaultCharset` key. The default is
     'defaultCharset': 'UTF-8'
 ```
 
+## relocateBuildTree
+That is, real object, library or exec directories are relocated to the
+current working directory or dir if specified. Ensure that it is full
+normalized patch ended with the directory separator.
+```javascript
+    'relocateBuildTree': '/home/user/project/build/'
+```
+
+## rootDir
+This option is to be used with relocateBuildTree above and cannot be
+specified alone. This option specifies the root directory for artifacts
+for proper relocation. Ensure that it is full normalized patch ended
```

Comment:
*path

Reply:
what is the difference between this path and the one specified in relocateBuildTree?

Reply:
Done. Based on the information from grpbuild documentation one is for object files and other is for project's artifacts.

Reply:
I suggest to add a link to https://docs.adacore.com/gprbuild-docs/html/gprbuild_ug/building_with_gprbuild.html#switches

Reply:
Link to the gprbuild documentation has been added